src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ruleNode.tsx

```diff
@@ -201,7 +295,9 @@ const RuleRow = styled('div')`
   display: flex;
   align-items: center;
   padding: ${space(1)};
+`;
 
+const RuleRowContainer = styled('div')`
```

Comment:
Do we need this container? Will this give the banner an offWhite bg?

Can we use a `<React.Fragment>` here?

Reply:
This container is used to give an offWhite bg to all items belonging to it. I believe children components with their own background-color won't have their colors overwritten. I didn't look into it, however this is how the container is being rendered: 
![image](https://user-images.githubusercontent.com/23180853/76579500-af25ab00-6489-11ea-96b6-a7890c960428.png)


I don't understand what you're suggesting here:
> Can we use a <React.Fragment> here?

Reply:
I was thinking you could replace the container with a [Fragment](https://reactjs.org/docs/fragments.html) but it may not be possible because of the CSS.

Reply:
Can we see what this looks like with the alerts above the rule? I think generally, alerts should be above.

Reply:
@billyvg since the message depends on the selection, isn't it weird to have the alert box appear on top of the selection?

Reply:
![image](https://user-images.githubusercontent.com/23180853/76650130-32451080-651f-11ea-979d-792ac0e23384.png)
You'll get this. Let me know which you you prefer @billyvg @adhiraj

Reply:
Also decided to trim the bottom margin for both variants
![image](https://user-images.githubusercontent.com/23180853/76650315-9071f380-651f-11ea-99e9-2e31a7330dc6.png)

Reply:
Hmm I think either works, but we may want it inside of the Rule Row maybe so it has the same padding?